docker/conda/environments/cuda11.5_dev.yml

```diff
@@ -29,10 +28,10 @@ dependencies:
     - ccache>=3.7
     - cmake=3.22
     - cuda-nvml-dev=11.4
```

Comment:
No `cuda-nvml-dev=11.5`?

Reply:
iirc, cuda-nvml-dev>=11.5,<11.6 wouldn't solve, but I can try again.

Reply:
we discussed offline that cuda-nvml-dev wouldn't solve when providing the `nvidia` conda channel, and instead we needed to add a specific `nvidia/label/cuda-11.5.2` channel.